docs/sections/userQuickStart.md

```diff
@@ -0,0 +1,131 @@
+<link rel="stylesheet" href="{{baseUrl}}/css/main.css">
+
+<include src="../common/header.md" />
+
+<div class="website-content">
+
+# User Quick Start
+
+## Requirement
+We expect users to have basic knowledge of the following:
+- [x] Markdown
+- [x] HTML & CSS
+- [x] npm (Node.js)
+- [x] Command-line Environment

```

Comment:
This list is a bit off-putting. May be no need for users to know html, css, node?